chaco/contour_poly_plot.py

```diff
@@ -9,6 +7,10 @@
 # is also available online at http://www.enthought.com/licenses/BSD.txt
 #
 # Thanks for using Enthought open source!
+
+""" Defines the ContourPolyPlot class.
+"""
+
```

Comment:
Was this intentional?

Reply:
Yes it was. I moved the module docstring after the copyright header.

Reply:
Ah I had meant to delete this in the reorg PR that is my mistake.  Per your review comment I had been deleting the other module docstrings of the stub files